packages/atdgen/atdgen.1.10.2/files/atdgen.install

```diff
@@ -0,0 +1,120 @@
+lib: [
```

Comment:
I think you don't need that file anymore. Jbuilder will generate a more precise one.